common/fixation_entry.go

```diff
@@ -0,0 +1,228 @@
+package common
+
+import (
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/common/types"
+	"github.com/lavanet/lava/utils"
+)
+
+// Set entry with full index. Full index is the entry index + version num suffix (like "bundle1_0")
```

Comment:
add some description in the beginning of the file explaining what it is

Reply:
done